src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -158,3 +167,12 @@ def test_osa_offline_reintegration(self):
         """
         # Perform reintegration testing with a pool
         self.run_offline_reintegration_test(1, True)
+
+    @skipForTicket("DAOS-6766, DAOS-6783")
+    def test_osa_offline_reintegration_server_stop(self):
+        """Test ID: DAOS-6748.
+        Test Description: Validate Online Reintegration with server stop
+        :avocado: tags=all,pr,daily_regression,hw,medium,ib2,osa
+        :avocado: tags=offline_reintegration_srv_stop,DAOS_5610
```

Comment:
We should follow the new agreed upon avocado tags norm now.
`:avocado: tags=all,pr,daily_regression`
`:avocado: tags=hw,medium,ib2`
`:avocado: tags=osa,offline_reintegration_srv_stop`
Removing DAOS_5610 as the ticket seem to be resolved.
If possible please do the same for other tests in this test file.

Reply:
Ok. Will do.

Reply:
removed.